tests/bluetooth/bsim_bt/bsim_test_notify/test_scripts/gatt_notify_any_any.sh

```diff
@@ -0,0 +1,10 @@
+
+#!/usr/bin/env bash
+# Copyright 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+simulation_id="gatt_notify_any_any" \
```

Comment:
What's the reasoning behind the "any_any" naming?

Reply:
Combination of which option the server and client uses. There are 4 cases for each of them (OPT_NONE, ENHANCED_ONLY, UNENHANCED_ONLY, mix of both "onlys").

"any" should be renamed to "none".